pkg/store/gcworker/gc_worker.go

```diff
@@ -751,62 +729,76 @@ func (w *GCWorker) runGCJob(ctx context.Context, safePoint uint64, concurrency g
 	})
 	metrics.GCWorkerCounter.WithLabelValues("run_job").Inc()
 
-	err := w.resolveLocks(ctx, safePoint, concurrency.v)
+	// ----------*--------------------*--------------------> time
+	//                                ^ Txn safe point (not synced)
+	//           ^ GC safe point (last value)
+
+	// The txn safe point should be guaranteed to be synchronized over all necessary components after
+	// `txnSafePointSyncWaitTime`. We wait for that time before performing any operation about the GC.
+	time.Sleep(txnSafePointSyncWaitTime)
```

Comment:
Let me check if I understand it correctly:
if other tidb instance cannot load the txn safe point within `txnSafePointSyncWaitTime` interval, 
it will reject any txn commit operation. The check commit ts step should fail.

Reply:
transaction committing is currently not checked, and only read operations are affected.
`txnSafePointSyncWaitTime` is 100s, but the max allowed interval between updating txn safe point is [90s](https://github.com/tikv/client-go/blob/74c0a811505babbb876c1f8e9c6f72b6fea12648/tikv/kv.go#L186C36-L186C60).

Before PR https://github.com/tikv/client-go/pull/1634 , this timeout was 99s.